### ISSUE TITLE
When I use version 1.0.0，it has error:  Uncaught SyntaxError: Unexpected token import

### ISSUE THREAD

**Issue by mh0625** [jbaysolutions/vue-grid-layout#40]
I have install vue-grid-layout of version 1.0.0，then I use
`import VueGridLayout from 'vue-grid-layout';`
to import vue-grid-layout，it has error：Uncaught SyntaxError: Unexpected token import.

the error code is 
vue-grid-layout/src/app.js   line 29
`import GridItem from './GridItem.vue';`

**Comment by gmsa** [comment]
Look at https://github.com/jbaysolutions/vue-grid-layout/blob/1.0.0/src/app.js for how to properly use the grid.

**Comment by mh0625** [comment]
I want to use GridLayout.vue. What should I do?

**Comment by Tinoooo** [comment]
Same here. Whenever I try to import a component (`import GridItem from 'vue-grid-layout/src/GridItem.vue';`), I get the following error.
```
{ Error: Parsing file C:\Users\xxx\node_modules\vue-grid-layout\src\GridItem.vue: Unexpected token (1:0)
    at Deps.parseDeps (C:\Users\xxx\node_modules\module-deps\index.js:481:28)
    at getDeps (C:\Users\xxx\node_modules\module-deps\index.js:414:40)
    at C:\Users\xxx\node_modules\module-deps\index.js:398:32
    at ConcatStream.<anonymous> (C:\Users\xxx\node_modules\concat-stream\index.js:36:43)
    at emitNone (events.js:91:20)
    at ConcatStream.emit (events.js:185:7)
    at finishMaybe (C:\Users\xxx\node_modules\concat-stream\node_modules\readable-stream\lib\_stream_writable.js:475:14)
    at endWritable (C:\Users\xxx\node_modules\concat-stream\node_modules\readable-stream\lib\_stream_writable.js:485:3)
    at ConcatStream.Writable.end (C:\Users\xxx\node_modules\concat-stream\node_modules\readable-stream\lib\_stream_writable.js:455:41)
    at DuplexWrapper.onend (C:\Users\xxx\node_modules\readable-stream\lib\_stream_readable.js:504:10)
```

Any idea on what is wrong here?

**Comment by gmsa** [comment]
Fixed by release 1.0.1, created a usage sample project [here](https://github.com/jbaysolutions/vue-grid-layout-sample-v1)
